packages/transaction-controller/src/TransactionController.ts

```diff
@@ -698,12 +698,12 @@ export class TransactionController extends BaseController<
       actionId,
     }: { estimatedBaseFee?: string; actionId?: string } = {},
   ) {
+    const { transactions } = this.state;
+
     if (gasValues) {
       validateGasValues(gasValues);
     }
-    const transactionMeta = this.state.transactions.find(
-      ({ id }) => id === transactionId,
-    );
+    const transactionMeta = transactions.find(({ id }) => id === transactionId);
```

Comment:
Should we use the `getTransaction` method rather than duplicate the `find`?

Reply:
Right, fixed in https://github.com/MetaMask/core/pull/1998/commits/0ee039e86a8ff7ab397943748f5103e0e6e5caba